lib/src/Base/Solver/ODESolverImplementation.cxx

```diff
@@ -85,6 +85,13 @@ FieldFunction ODESolverImplementation::getTransitionFunction() const
   return transitionFunction_;
 }
 
+Point ODESolverImplementation::evaluate(const Scalar t, const Point & inP) const
+{
+  Field X(Mesh(Sample(1, Point(1, t))), Sample(1, inP));
+  Field Y(transitionFunction_(X));
+  return Y.getValues()[0];
+}
```

Comment:
As said previously, IMO transitionFunction_ should be a ParametricFunction instead of a FieldFunction, so that we do not have to create so many objects for each evaluation.